vmdb/app/models/ems_amazon.rb

```diff
@@ -24,10 +24,21 @@ def description
   # Connections
   #
 
-  def self.raw_connect(access_key_id, secret_access_key, service = nil, region = nil, proxy_uri = nil)
+  def self.raw_connect(access_key_id, secret_access_key, service, region = nil, proxy_uri = nil)
     service   ||= "EC2"
```

Comment:
It turns out this is indeed still needed; the parameter is always supplied, but it sometimes gets passed as `nil`.